CHANGELOG.md

```diff
@@ -9,19 +9,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
--   [#39](https://github.com/green-code-initiative/ecoCode-javascript/issues/39) Add rule `@ecocode/avoid-brightness-override` (EC522)
--   [#41](https://github.com/green-code-initiative/ecoCode-javascript/pull/41) Add rule `@ecocode/no-torch` (EC530)
+-   [#39](https://github.com/green-code-initiative/ecoCode-javascript/issues/39) Add rule `@creedengo/avoid-brightness-override` (GCI522)
```

Comment:
why don't you upgrade the link also ?

Reply:
I have updated links to new repository in the **Unrealeased** section of the CHANGELOG in 8c307ebd5385d74a7b4f775a613265bc7a2c2ff8.
I propose to do not touch the rest of the file because versions have already been released.